### DISCUSSION TITLE
Automatically catch email if customer says email as first message

### DISCUSSION THREAD

**Discussion by Christilut** [chatwoot/chatwoot#3950] [Feature Requests]
This happens quite often:
![image](https://user-images.githubusercontent.com/3471363/153172543-d0e34948-6e45-4b45-9df1-8deb9de8eaa5.png)
With the first first message after that being `foo@example.com`.

So instead of using the email box, the customer just types it. Then I have to copy the email and fill it in their details so my messages will reach them by email.

My suggestion is to automatically rectify this behavior when the first message after the "Get notified by email" is an email address (and nothing more).

## In short
1. User sees "Get notified by email" but does not fill it in
2. User chats their email address
3. Chatwoot recognizes this and adds that email to the user details (same as when the user would have filled in "Get notified by email"

## Extra thoughts
This could even be done client side by just calling the "Get notified by email" with the first email chat message.
